source/Launcher/Launcher.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Microsoft.Win32;
+using System;
+using System.Diagnostics;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace Launcher
+{
+    public partial class BannerlordCoopLauncher : Form
+    {
+        private bool mouseDown;
+        private Point lastLocation;
+        public BannerlordCoopLauncher()
+        {
+            InitializeComponent();
+        }
+
+        /// <summary>
+        /// Check if MB is installed (through the registry) - Only works in x64.
+        /// </summary>
+        /// <returns>Full path where MB is installed</returns>
+        private string GetBannerLordInstalledPath()
```

Comment:
Since this is copy pasta, lets pull this out into some status util for both classes to use. If for some reason this changes in future releases, we don't want to have to change it twice